### ISSUE TITLE
Master fails to build

### ISSUE THREAD

**Issue by vasily-kirichenko** [JetBrains/fsharp-support#26]
```
Build with surface heuristics started at 12:19:28
Use build tool: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\MSBuild.exe
0>------- Project started: FSharp.Psi
0>src\Impl\DeclaredElement\FSharpConstructorBase.cs(32,26): Warning CS0108: 'FSharpConstructorBase<TDeclaration>.ReturnKind' hides inherited member 'FSharpMemberBase<TDeclaration>.ReturnKind'. Use the new keyword if hiding was intended.
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\AbstractTypeDeclaration.cs(13,41): Error CS0535: 'AbstractTypeDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\AbstractTypeDeclaration.cs(13,41): Error CS0535: 'AbstractTypeDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\ClassDeclaration.cs(13,41): Error CS0535: 'ClassDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\ClassDeclaration.cs(13,41): Error CS0535: 'ClassDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>src\Impl\Tree\FSharpTypeElementDeclarationBase.cs(14,91): Error CS0535: 'FSharpTypeElementDeclarationBase' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>src\Impl\Tree\FSharpTypeElementDeclarationBase.cs(14,91): Error CS0535: 'FSharpTypeElementDeclarationBase' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\EnumDeclaration.cs(13,41): Error CS0535: 'EnumDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\EnumDeclaration.cs(13,41): Error CS0535: 'EnumDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\InterfaceDeclaration.cs(13,41): Error CS0535: 'InterfaceDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\InterfaceDeclaration.cs(13,41): Error CS0535: 'InterfaceDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\ModuleAbbreviation.cs(13,41): Error CS0535: 'ModuleAbbreviation' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\ModuleAbbreviation.cs(13,41): Error CS0535: 'ModuleAbbreviation' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\OtherSimpleTypeDeclaration.cs(13,41): Error CS0535: 'OtherSimpleTypeDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\OtherSimpleTypeDeclaration.cs(13,41): Error CS0535: 'OtherSimpleTypeDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\NestedModuleDeclaration.cs(13,41): Error CS0535: 'NestedModuleDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\NestedModuleDeclaration.cs(13,41): Error CS0535: 'NestedModuleDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\ExceptionDeclaration.cs(13,41): Error CS0535: 'ExceptionDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\ExceptionDeclaration.cs(13,41): Error CS0535: 'ExceptionDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\StructDeclaration.cs(13,41): Error CS0535: 'StructDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\StructDeclaration.cs(13,41): Error CS0535: 'StructDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\TypeAbbreviationDeclaration.cs(13,41): Error CS0535: 'TypeAbbreviationDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\TypeAbbreviationDeclaration.cs(13,41): Error CS0535: 'TypeAbbreviationDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\RecordDeclaration.cs(13,41): Error CS0535: 'RecordDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\RecordDeclaration.cs(13,41): Error CS0535: 'RecordDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\NestedTypeUnionCaseDeclaration.cs(13,41): Error CS0535: 'NestedTypeUnionCaseDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\NestedTypeUnionCaseDeclaration.cs(13,41): Error CS0535: 'NestedTypeUnionCaseDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\UnionDeclaration.cs(13,41): Error CS0535: 'UnionDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\UnionDeclaration.cs(13,41): Error CS0535: 'UnionDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\ObjectTypeDeclaration.cs(13,41): Error CS0535: 'ObjectTypeDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\ObjectTypeDeclaration.cs(13,41): Error CS0535: 'ObjectTypeDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\TopLevelModuleDeclaration.cs(13,41): Error CS0535: 'TopLevelModuleDeclaration' does not implement interface member 'ITypeDeclaration.NestedTypeDeclarationsEnumerable'
0>obj\Debug\net461\ParserGen\Psi\FSharp\Impl\TopLevelModuleDeclaration.cs(13,41): Error CS0535: 'TopLevelModuleDeclaration' does not implement interface member 'ITypeDeclarationHolder.TypeDeclarationsEnumerable'
0>------- Project finished: FSharp.Psi. Succeeded: False. Errors: 32. Warnings: 1
Build completed in 00:00:02.879
Build failed at 12:19
```

**Comment by auduchinok** [comment]
@vasily-kirichenko Thanks, it's due to another recent change to the SDK. I'll fix it shortly.

**Comment by auduchinok** [comment]
Fixed in 631fcb14dbd48ed92a237f428bca66dbc50dac8a.
